easybuild/tools/build_log.py

```diff
@@ -63,7 +64,19 @@ class EasyBuildError(LoggedException):
 
     def __init__(self, msg, *args):
         """Constructor: initialise EasyBuildError instance."""
-        msg = msg % args
+        # figure out where error was raised from
+        # current frame: this constructor, one frame above: location where this EasyBuildError was created/raised
+        frameinfo = inspect.getouterframes(inspect.currentframe())[1]
```

Comment:
missing remark by @stdweird: this is quite generic, so let's move it to vsc-base instead, and add support to LoggedException to disable this optionally (but have it enabled by default)